packages/expect/src/jasmine_utils.js

```diff
@@ -51,6 +51,7 @@ function asymmetricMatch(a, b) {
   }
 
   if (asymmetricA) {
+    // $FlowFixMe 0.85
```

Comment:
mind commenting on why instead of just `0.85`?

Reply:
just did. Flow infers `a` to be a number, because of a strict comparison to `0` in line 105. Can't blame it for not being able to infer it correctly based on this fishy switch statement though.